registration/tests/test_views.py

```diff
@@ -556,6 +591,44 @@ def test_modifying_design_fields_sends_email(self):
             self._register(modified)
             self.assertEqual(len(mail.outbox), original_emails + 1)
 
+    def test_preserve_empty_privacy_url(self):
```

Comment:
We can make the users leave empty values for all instances, not only the existing ones. This will reduce overall complexity.

Reply:
It will, but it will also add a privacy policy to existing instances without the owner's knowledge, which doesn't feel like a good experience - they should at least have the opportunity to review the privacy policy and provide their own if they don't like the one provided by default.